packages/router/utils.ts

```diff
@@ -1505,12 +1505,12 @@ export const redirectDocument: RedirectFunction = (url, init) => {
  * @private
  * Utility class we use to hold auto-unwrapped 4xx/5xx Response bodies
  */
-export class ErrorResponse {
+export class ErrorResponseImpl {
   status: number;
   statusText: string;
   data: any;
-  error?: Error;
-  internal: boolean;
+  private error?: Error;
+  private internal: boolean;
```

Comment:
These should have been private all along, but since the entire class was a "private" implementation detail I think I glossed over the field specifiers.  Marking them private ensures the type below doesn't include them.